modules/@ergonode/channels/src/config/extends.js

```diff
@@ -58,10 +58,10 @@ export default {
 
             exportStatuses.forEach((notification) => {
                 const {
-                    processed, items,
+                    status,
                 } = notification;
 
-                if (processed < items) {
+                if (status === 'PRECESSED') {
```

Comment:
should this value not be in a constant variable?

Reply:
Nah, product statuses are not defined yet. It's going to be improved soon